packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerCard/AddPrayerCard.js

```diff
@@ -17,19 +16,18 @@ const StyledView = styled(({ theme }) => ({
   paddingVertical: theme.sizing.baseUnit * 4,
   borderBottomLeftRadius: theme.sizing.baseUnit,
   backgroundColor: theme.colors.paper,
-}))(FlexedView);
+  justifyContent: 'center',
+}))(View);
 
 const StyledBodyText = styled(({ theme }) => ({
   paddingVertical: theme.sizing.baseUnit,
 }))(BodyText);
 
-const StyledCardContent = styled({
-  flex: 1,
-})(CardContent);
+const StyledCardContent = styled({})(CardContent);
 
 const StyledFlexedView = styled(({ theme }) => ({
```

Comment:
```suggestion
const StyledView = styled(({ theme }) => ({
```

Reply:
I agree that "Flexed" should probably come out of the name, but we'll need a different name since there is already another `StyledView`. Maybe `StyledButtonView` since this `View` holds a `Button`?

Reply:
Agreed